packages/components/src/assets/ClockClockwise.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import type { FC } from 'react'
+
+// Icon from Phosphor Icons
+const ClockClockwise: FC = () => (
```

Comment:
How about just calling it clockwise? 🤔

Reply:
it's the name used by [Phosphor Icons](https://phosphoricons.com/): 
<img width="500px" alt="Screenshot 2023-02-27 at 15 25 41" src="https://user-images.githubusercontent.com/11613011/221650970-ff1055c0-949f-4158-985e-89b124a9f7cc.png">

Reply:
My 2 cents, I think `ClockClockwise` is more precise, even though it looks weird 😅 . We could have a `ClockCounterClockwise` icon as well.